ui/lobby/src/view/presets.ts

```diff
@@ -43,11 +44,19 @@ function presetPerf(p: Preset): Perf {
   return p.timeMode == 2 ? 'correspondence' : clockToPerf(p.lim * 60, p.byo, p.inc, p.per);
 }
 
+function presetLabel(p: Preset, perf: Perf) {
+  const ja = useJapanese();
+  return ja && !p.lim && p.byo ? `${p.byo}秒秒読み` :
+    ja && p.lim && !p.byo ? `${p.lim == 3 ? '３' : p.lim == 5 ? '５' : p.lim}分切れ負け` :
```

Comment:
I don't know how to dynamically generate the Japanese Arabic double-wide numerals, if that matters.

Reply:
Thanks, I've added function `fullWidthNumber` with the same body.